frontend/views/containers/chatroom/MessageBase.vue

```diff
@@ -18,57 +18,38 @@
           span.has-text-1 {{ humanDate(datetime, { hour: 'numeric', minute: 'numeric' }) }}
 
       slot(name='body')
-        p.c-replying.custom-markdown-content(
-          v-if='replyingMessage'
-          @click='$emit("reply-message-clicked")'
-        )
-          template(v-for='(objReplyMessage, index) in replyMessageObjects')
-            span(
-              v-if='isText(objReplyMessage)'
-              v-safe-html:a='objReplyMessage.text'
-            )
-            span.c-member-mention(
-              v-else-if='isMemberMention(objReplyMessage)'
-              :class='{"c-mention-to-me": objReplyMessage.toMe}'
-            ) {{ objReplyMessage.text }}
-            span.c-channel-mention(
-              v-else-if='isChannelMention(objReplyMessage)'
-              :tabindex='objReplyMessage.disabled ? undefined : 0'
-              :class='{ "is-disabled": objReplyMessage.disabled }'
-              @click='navigateToChatRoom(objReplyMessage)'
-              @keyup.enter='navigateToChatRoom(objReplyMessage)'
-            )
-              i(:class='"icon-" + objText.icon')
-              span {{ objText.text }}
+        template(v-if='replyingMessage')
+          render-message-with-markdown(
+            v-if='shouldRenderMarkdown'
+            :isReplyingMessage='true'
+            :text='replyingMessage'
+            @click='$emit("reply-message-clicked")'
+          )
+          render-message-text.c-replying(
```

Comment:
I think we can simplify these codes using `:is` attribute just like how we do in `ChatMain.vue`.
```
...

component(
  :is='messageType(message)'

...

computed () {
  messageType (message) {
    return {
      [MESSAGE_TYPES.NOTIFICATION]: 'message-notification',
      [MESSAGE_TYPES.INTERACTIVE]: 'message-interactive',
      [MESSAGE_TYPES.TEXT]: 'message',
      [MESSAGE_TYPES.POLL]: 'message-poll'
    }[message.type]
  }

...

```

Reply:
@Silver-IT Good point, but let me modify the logic that way when the number of the options becomes `>2` in the future. Currently only two components are used here and I feel like `v-if/v-else` pair would make it tidy enough, readability-wise.

Reply:
Even though, we use two components, I personally prefer to use `:is` attribute because we can use it in two places; one for original message and one for replying message. Of course, it's not a big issue, and just shared my opinion.

I am going to approve this PR, since nothing wrongs I found. But I would like to keep this conversation, and to know how @taoeffect thinks about it.

Reply:
> Of course, it's not a big issue, and just shared my opinion.

Yeah, I think it's not a big of a deal now either way and I still like to keep it as `v-if/v-else`, as they look more straightforward to read. But if you really feel like, you can go ahead and create an issue for it and work on adding `<component :is='...'>` as a follow-up work of this PR.